org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -601,6 +610,28 @@ SsoApplicationManager:
         ]
       }
 
+# Role for a user that can access BedRock in the Synapse LLM prod account
+SSoLLmDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.7/templates/SSO/aws-sso.njk
+  StackName: !Sub '${resourcePrefix}-${appName}-llmdeveloper'
+  StackDescription: 'Role used by LLM Developer group within Synapse LLM prod account'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref SynapseLlmProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref SynapseLLmProdDeveloperGroup
+    permissionSetName: 'llm-developer'
+    managedPolicies:
```

Comment:
At a minimum you will need to give developers some type of general READ access to the account otherwise they won't be able to view anything.  Basically you'll need to duplicate some of the permissions that are given to [genernal SsoDeveloper](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L394-410).  You'll definately want to add the `Deny` policy.

Reply:
Will add deny from SSoDeveloper... Is it not the case that we don't want llm devs to create any resources beyond Bedrock? I think they are only supposed to tinker with the prompts and manage Bedrock agents.

Reply:
ok, we can try with this however i don't think it's enough for an LLM developer.

Reply:
ok, we can try with this scoped policy but i don't think it will be enough.